ReleaseNotes-2.9.8

```diff
@@ -0,0 +1,23 @@
+#
+# Open Build Service 2.9.8
+#
+
+Please read the README.SETUP file for initial installation
+instructions or use the OBS Appliance from
+
+  http://openbuildservice.org/download
+
+There is also an install medium which installs OBS on hard disk now.
+
+dist/README.UPDATERS file has information for updaters.
+
+OBS Appliance users who have setup their LVM can just replace
+their appliance image without data loss. The migration will
+happen automatically.
+
+Bugfixes
+========
+
+Frontend:
+  * Security update for gem nokogiri (CVE-2020-7595)
```

Comment:
Ne need to mention this, we were not effected by this CVE as we use the system libxml.